engine/executor/dm/api.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dm
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/pingcap/errors"
+
+	dmpkg "github.com/pingcap/tiflow/engine/pkg/dm"
+)
+
+// QueryStatus implements the api of query status request.
+func (t *dmWorker) QueryStatus(ctx context.Context, req *dmpkg.QueryStatusRequest) *dmpkg.QueryStatusResponse {
```

Comment:
```suggestion
func (w *dmWorker) QueryStatus(ctx context.Context, req *dmpkg.QueryStatusRequest) *dmpkg.QueryStatusResponse {
```

IDE says receiver name should be consistent